src/components/AddSample/AddSample.jsx

```diff
@@ -0,0 +1,378 @@
+
+import { useDispatch, useSelector } from 'react-redux';
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+
+// material ui imports 
+import { makeStyles } from '@material-ui/core/styles';
+import { TextField, 
+  Button, MenuItem,
+  FormHelperText, FormControl, Select, Typography } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  inputs: {
+   margin: theme.spacing(2),
+  },
+  formControl: {
+    margin: theme.spacing(2),
+    minWidth: 180,
+  },
+}));
+
+
+function AddSample() {
+  // material ui
+  const classes = useStyles();
+
+  // set stuff so we can use them
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  // get state from redux store
+  const user = useSelector(store => store.user);
+
+  // local states for input fields
+  const [ingredientName, setName] = useState('');
+  const [lotNumber, setLotNumber] = useState('');
+  const [format, setFormat] = useState('');
+  const [ingredientAmount, setAmount] = useState('');
+  const [ingredientUnit, setIngredientUnit] = useState('');
+  const [purity, setPurity] = useState('');
+  const [dateManufactured, setDate] = useState('');
+  const [extractionMethod, setMethod] = useState('');
+  const [city, setCity] = useState('');
+  const [state, setState] = useState('');
+  const [country, setCountry] = useState('');
+  const [harvestDate, setHarvestDate] = useState('');
+  const [sustainability, setSustainability] = useState('');
+  const [cropStrain, setCropStrain] = useState('');
+
+  // functions
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log("form submit")
+  };
+
+  // on shipping button click
+  const shipping = (event) => {
+    event.preventDefault();
+    console.log('shipping info');
+
+    if (
+      !ingredientName ||
+      !lotNumber ||
+      !format ||
+      !ingredientAmount ||
+      !dateManufactured ||
+      !extractionMethod
+    ) {
+      alert("Please complete required inputs");
```

Comment:
I'm not able to get past this validation step. `dateManufactured` is always an empty string